lnwallet/channel.go

```diff
@@ -1023,11 +1023,10 @@ type commitmentChain struct {
 	startingHeight uint64
 }
 
-// newCommitmentChain creates a new commitment chain from an initial height.
-func newCommitmentChain(initialHeight uint64) *commitmentChain {
+// newCommitmentChain creates a new commitment chain.
+func newCommitmentChain() *commitmentChain {
 	return &commitmentChain{
-		commitments:    list.New(),
-		startingHeight: initialHeight,
+		commitments: list.New(),
```

Comment:
Where do we set the new starting height of a commitment chain? Don't see why this change is necessary atm. There're other spots where we depend on the height being set properly.

Reply:
We never read it, we just look at the height of the commitment in the chain.

Agreed that the change is not directly relevant to this PR, I just noticed when transforming the `ChanSync` method to not being dependent on the chain heights.

Removed and made a separate PR: https://github.com/lightningnetwork/lnd/pull/2206